File: baekjoon/string/14425.py
import sys
from typing import List, Set
input = sys.stdin.readline


def solution(refs: Set[str], targets: List[str]) -> int:
    count = 0

    for target in targets:
        if target in refs:
            count += 1

    return count


if __name__ == "__main__":
    N, M = map(int, input().split())
    S, R = set(), []

    for _ in range(N):
        S.add(input())

    for _ in range(M):
        R.append(input())

    print(solution(S, R))

File: baekjoon/string/14425.js
const solution = (refs, targets) => {
  const S = new Set(refs);

  let count = 0;
  for (const target of targets) {
    if (S.has(target)) count += 1;
  }

  return count;
};

function main() {
  const fs = require('fs');
  const input = fs.readFileSync('./baekjoon/string/14425.txt').toString().trim().split('\n');

  const N = +input[0].split(' ')[0];
  console.log(solution(input.slice(1, N + 1), input.slice(N + 1)));
}

if (require.main === module) {
  main();
}
